dbgpt/configs/model_config.py

```diff
@@ -212,11 +212,12 @@ def get_device() -> str:
     # Common HTTP embedding model
     "proxy_http_openapi": "proxy_http_openapi",
     "proxy_ollama": "proxy_ollama",
+    "proxy_tongyi": "proxy_tongyi",
     # Rerank model, rerank mode is a special embedding model
     "bge-reranker-base": os.path.join(MODEL_PATH, "bge-reranker-base"),
     "bge-reranker-large": os.path.join(MODEL_PATH, "bge-reranker-large"),
     # Proxy rerank model
-    "rerank_proxy_http_openapi": "rerank_proxy_http_openapi",
+    "rerank_proxy_http_openapi": "",
```

Comment:
Is there a problem here? If not, this place does not need to be modified~

Reply:
I have fix the problem. revert rerank_proxy_http_openapi config